owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -275,6 +288,13 @@ abstract class DrawerActivity : ToolbarActivity() {
         sendEmail(email = feedbackMail, subject = feedback)
     }
 
+    private fun openDrawerLink() {
+        goToUrl(url = resources.getString(R.string.drawer_link))
+    }
+
+    private fun isDrawerLinkEnabled() =
+        resources.getString(R.string.drawer_link_label).isNotEmpty() && resources.getString(R.string.drawer_link).isNotEmpty()
```

Comment:
This check probably is done somewhere else, but we can introduce it anyway in case we receive `" "` 👍 
```suggestion
        resources.getString(R.string.drawer_link_label).isNotBlank() && resources.getString(R.string.drawer_link).isNotBlank()
```